<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::HierarchyViewer</class>
 <widget class="QWidget" name="sf::HierarchyViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HierarchyViewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tbExpandAll"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbCollapseAll"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch">
       <property name="placeholderText">
        <string>Search by name...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="indentation">
      <number>16</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tbExpandAll</tabstop>
  <tabstop>tbCollapseAll</tabstop>
  <tabstop>tbEdit</tabstop>
  <tabstop>leSearch</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
